encoding/kzg/kzg.go

```diff
@@ -22,45 +22,47 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
+//go:build !bignum_pure && !bignum_hol256
+// +build !bignum_pure,!bignum_hol256
+
 package kzg
 
 import (
-	bls "github.com/Layr-Labs/eigenda/pkg/kzg/bn254"
+	"github.com/Layr-Labs/eigenda/encoding/fft"
+	"github.com/consensys/gnark-crypto/ecc"
+	"github.com/consensys/gnark-crypto/ecc/bn254"
+	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
 )
 
-// invert the divisor, then multiply
-func polyFactorDiv(dst *bls.Fr, a *bls.Fr, b *bls.Fr) {
-	// TODO: use divmod instead.
-	var tmp bls.Fr
-	bls.InvModFr(&tmp, b)
-	bls.MulModFr(dst, &tmp, a)
+type KZGSettings struct {
```

Comment:
This object looks possibly extraneous to me. Maybe something to cleanup in a future refactor.